griffon/commands/queries.py

```diff
@@ -361,14 +361,19 @@ def get_product_contain_component(
         params.pop("flaw_mode")
         params.pop("affect_mode")
         if component_name:
-            q = query_service.invoke(core_queries.products_containing_component_query, params)
+            q = query_service.invoke(
```

Comment:
Note: I like these progress bar notes since the user is well informed on what happens however in the future I would be for unification of the progress bar approach across the project because currently we do have a `progress_bar` decorator and this direct context manager approach side by side used.

Reply:
ya, we have a general decorator which we do not use in this specific operation - that is because click's progress bar is broken (or more like undocumented) ... will raise a task to harmonise this in future work.